cmd/argocd/commands/configuration.go

```diff
@@ -0,0 +1,42 @@
+package commands
+
+import (
+	"fmt"
+	"strconv"
+
+
+	"github.com/spf13/cobra"
+
+	argocdclient "github.com/argoproj/argo-cd/v2/pkg/apiclient"
+	"github.com/argoproj/argo-cd/v2/util/errors"
+	"github.com/argoproj/argo-cd/v2/util/localconfig"
+)
+
+// NewConfigurationCommand returns a new instance of an `argocd configuration` command
+func NewConfigurationCommand(globalClientOpts *argocdclient.ClientOptions) *cobra.Command {
+	var promptsEnabled bool
+	command := &cobra.Command{
+		Use:   "configuration",
+		Short: "Manage local configuration",
+		Example: `# Enable optional interactive prompts
+argocd configuration --prompts-enabled
+argocd configuration --prompts-enabled=true
+
+# Disable optional interactive prompts
+argocd configuration --prompts-enabled=false`,
+		Run: func(c *cobra.Command, args []string) {
+			localCfg, err := localconfig.ReadLocalConfig(globalClientOpts.ConfigPath)
+			errors.CheckError(err)
+
+			localCfg.PromptsEnabled = promptsEnabled
+
+			err = localconfig.WriteLocalConfig(*localCfg, globalClientOpts.ConfigPath)
+			errors.CheckError(err)
+
+			fmt.Println("Successfully updated the following configuration settings:")
+			fmt.Printf("prompts-enabled: %v", strconv.FormatBool(localCfg.PromptsEnabled))
+		},
+	}
+	command.Flags().BoolVar(&promptsEnabled, "prompts-enabled", false, "Enable (or disable) optional interactive prompts")
```

Comment:
Lets change it to pointer and prompts, so we can verify existance of this flag

Reply:
As far as I can tell, the package pflag (https://github.com/spf13/pflag) doesn't actually support pointers for boolean flags

The alternative approach I've identified is for the default `--prompts-enabled` and `--force-prompts-enabled` flag values to be read from local configuration